apollo-integration/src/test/graphql/com/apollographql/android/impl/normalizer/HeroNameResponse.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
seems identical to `EpisodeHeroNameResponse.json` confirming that both are needed